packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -614,8 +615,13 @@ class _FocusState extends State<Focus> {
 
   void _handleAutofocus() {
     if (!_didAutofocus && widget.autofocus) {
-      FocusScope.of(context).autofocus(focusNode);
       _didAutofocus = true;
+      // Waiting for inactive nodes to be disposed.
+      SchedulerBinding.instance!.addPostFrameCallback((_) {
+        if (mounted) {
+          FocusScope.of(context).autofocus(focusNode);
```

Comment:
Is it possible for this `focusNode` to be disposed?

Reply:
It's probably not possible, since the widget hosting the node is mounted.

Reply:
Oh sorry I thought this line was in `FocusScopeState`. Although very unlikely but it still can happen if someone schedules a `focsusNode.dispose` in a postframe callback right?

Reply:
I think the life cycle of `focusNode` and `_FcusState` should be consistent